### DISCUSSION TITLE
Thank you - and note for all to remove '&' from any IMDB title lists to avoid sync issues

### DISCUSSION THREAD

**Discussion by Donostio** [cecobask/imdb-trakt-sync#27] [General]
Thanks again for creating this repository and for your help and advice getting it working for me. Please let me know how I can pay for you to have a beer as small token of my gratitude. I changed the secret IMDB list ID from 'all' to import each list at a time and isolated the issue with one particular list. It was the longest list but it tunrned out it wasn't exceeding the tract API limit (the error message must have related to exceeding the failed attempts limits). I noticed that the title included '&' and on a hunch I changed it to 'and' and then it worked fine so I've been able to put all the lists back into the secreet as comma separated values and they're all working brilliantly now. Thank you so much.

**Comment by cecobask** [comment]
Hi @Donostio,

Glad I could help you! The best way to support me is via the [Sponsor](https://github.com/sponsors/cecobask) button in the repository.

If all your lists have a valid name now, you won't have to provide the list IDs explicitly anymore. I've been using the syncer with the `all` option since the beginning and haven't experienced problems.

I'll investigate the problem with special characters (e.g. `&`) in list names soon and fix the problem or update the documentation with instructions around such cases.

**Reply by Donostio** [reply]
Thanks again. Following your advice I've set it up afresh as a fork (I'm not sure how I ended up cloning the main repository instead of forking it) so it's easier to sync with any changes to your man repository. All worked fine first time this time. 

I will stick with comma seperated titles of the lists in the secret list ID - I want to sync a selection of my IMDB lists only becasue I have manually created 'watched' lists that I don't need it Trakt and they would also take me over the ten lists limit in my free Trakt account and create a blockage.

I have sent you a virtual beer using the sponsor button. Sláinte!

**Reply by cecobask** [reply]
Thank you, @Donostio! ❤️ 

---

UPDATE:

I investigated the problem with special characters in IMDb list names (e.g. `&`).
This was an oversight on my part - the order of operations when building the Trakt list slug/s was incorrect.
If you sync your fork with the latest changes, you will be able to create lists with special characters in them 👍 
Reference commit: 8acd897753500284368b486d471899c7b1d4f6d3
